lib/ParserHelpers.js

```diff
@@ -0,0 +1,52 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Tobias Koppers @sokra
+*/
+"use strict";
+const BasicEvaluatedExpression = require("./BasicEvaluatedExpression");
+const ConstDependency = require("./dependencies/ConstDependency");
+const UnsupportedFeatureWarning = require("./UnsupportedFeatureWarning");
+
+const ParserHelpers = exports;
+
+ParserHelpers.addParsedVariableToModule = function(parser, name, expression) {
+	if(!parser.state.current.addVariable) return false;
+	var deps = [];
+	parser.parse(expression, {
+		current: {
+			addDependency: function(dep) {
+				dep.userRequest = name;
+				deps.push(dep);
+			}
+		},
+		module: parser.state.module
+	});
+	parser.state.current.addVariable(name, expression, deps);
+	return true;
+};
+
+ParserHelpers.toConstantDependency = function toConstantDependency(value) {
+	return function(expr) {
```

Comment:
could you name these anonymous function? It better for stacktraces and profiling...